### PR TITLE
Refactor chat-plugins to use template strings

### PR DESCRIPTION
**Pull request by panpawn**
I would also like to thank Slayer95 for showing me a tool to do this for me; I was going to do it all out by hand...

### CODE REVIEW

**Comment by panpawn** [comment]
I would like to note this isn't "finished"... "yet".

**Comment by Zarel** [comment]
And that conflict is why I said "ugh" when I saw this pullreq...

Sorry! My refactor was really high-priority.

File: chat-plugins/hangman.js [hunk 1]
```diff
@@ -130,33 +130,33 @@ class Hangman extends Rooms.RoomGame {
...
+		if (this.hint) output += `<div>(Hint: ${Tools.escapeHTML(this.hint)})</div>`;
+		output += `<p style="font-weight:bold;font-size:12pt;letter-spacing:3pt">${wordString}</p>`;
 		if (this.guesses.length) {
 			if (this.letterGuesses.length) {
-				output += 'Letters: ' + this.letterGuesses.map(g =>
-					'<strong' + (g[1] === '1' ? '' : ' style="color: #DBA"') + '>' + Tools.escapeHTML(g[0]) + '</strong>'
-				).join(', ');
+				output += `Letters: ${this.letterGuesses.map(g =>
+    '<strong' + (g[1] === '1' ? '' : ' style="color: #DBA"') + '>' + Tools.escapeHTML(g[0]) + '</strong>'
+).join(', ')}`;
```
Comment (Zarel) [inline chat-plugins/hangman.js]: Incidentally, this multiline stuff is definitely not done the right way...

Comment (panpawn) [inline chat-plugins/hangman.js]: The spacing looks off, is that what you mean?

Comment (Zarel) [inline chat-plugins/hangman.js]: The spacing is off because it's using multiline template strings. Do not use multiline template strings.

Comment (Zarel) [inline chat-plugins/hangman.js]: Actually if we could make an eslint rule for that, that'd be nice.

**Comment by panpawn** [comment]
I decided I'm going to do this in peicemeal

**Comment by panpawn** [comment]
Also, I haven't refactored ``chat-plugins/wifi.js`` on this pull request because:
>@bumbadadabum: also panpawn, I'm working on chat-plugins/wifi.js and will convert that to template strings myself

**Comment by panpawn** [comment]
This is still a work-in-progress pull request. What I am aware of that still needs to be done on this is:
- Remove ternary operators and other serious string-processing outside of new template strings

**Comment by Zarel** [comment]
This may be a bit early; I'm still working out code style standards.

**Comment by Slayer95** [comment]
imo all of this is very fair use of template strings

**Comment by Zarel** [comment]
imo I have opinions regarding template strings and don't like some of what's going on here

**Comment by Zarel** [comment]
Closing this. Template strings should probably be refactored piecemeal, in accordance with the new `CONTRIBUTING.md` spec.

**Pull request closed without merging**